KubeArmor/enforcer/bpflsm/enforcer.go

```diff
@@ -387,6 +387,10 @@ func (be *BPFEnforcer) TraceEvents() {
 			log.Source = string(bytes.Trim(event.Data.Source[:], "\x00"))
 			log.ProcessName = log.Source
 		}
+		// fallback logic if we don't receive resource from BuildLogBase()
+		if len(log.Resource) == 0 {
+			log.Resource = log.ProcessName
+		}
```

Comment:
Please add a check to do it only for Operation Process, or do it in the mon.SecurityBprmCheck case itself